butte/src/parser_macros.rs

```diff
@@ -12,31 +12,34 @@ macro_rules! single_value {
     };
 }
 
-#[test]
-fn test_single_value() {
-    use crate::types::*;
+#[cfg(test)]
+mod single_value_tests {
+    #[test]
+    fn test_single_value() {
+        use crate::types::*;
 
-    let val = single_value!(1);
-    assert_eq!(val, Single::Scalar(Scalar::Integer(1)));
+        let val = single_value!(1);
+        assert_eq!(val, Single::Scalar(Scalar::Integer(1)));
 
-    let val = single_value!(1.0);
-    assert_eq!(val, Single::Scalar(Scalar::Float(1.0)));
+        let val = single_value!(1.0);
+        assert_eq!(val, Single::Scalar(Scalar::Float(1.0)));
 
-    let val = single_value!(true);
-    assert_eq!(val, Single::Scalar(Scalar::Boolean(true)));
+        let val = single_value!(true);
+        assert_eq!(val, Single::Scalar(Scalar::Boolean(true)));
 
-    let val = single_value!(false);
-    assert_eq!(val, Single::Scalar(Scalar::Boolean(false)));
+        let val = single_value!(false);
+        assert_eq!(val, Single::Scalar(Scalar::Boolean(false)));
 
-    let val = single_value!("a");
-    assert_eq!(val, Single::StringConstant("a"));
+        let val = single_value!("a");
+        assert_eq!(val, Single::StringConstant("a"));
+    }
 }
 
 #[macro_export]
 macro_rules! namespace {
-    ($path:path) => {
+    ($path:expr) => {
```

Comment:
Why is path an expr and not a path?

Reply:
I think this is a remnant from a different approach I had taken, and I think it should actually be a path.